app-web/src/components/Search/index.js

```diff
@@ -59,7 +59,7 @@ export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest })
         type="text"
         query={query}
         aria-label={ARIA_LABEL_SEARCH_INPUT}
-        value={terms}
+        value={terms.useState}
```

Comment:
where `.useState` coming from? `terms` should be fine on its own, I think what you are looking for is to initially set terms with a query if its available. In that case, you could use a `useEffect(() => {...}, [])` to accomplish this.